fixtures/minimal/http_pass_single.yaml

```diff
@@ -11,6 +11,14 @@ spec:
       name: http-deprecated-endpoint
     - name: http-minimal-check
       url: https://httpbin.demo.aws.flanksource.com/status/200
+      metrics:
```

Comment:
can we also add examples for junit e.g. number of tests, passing, failing, time

Reply:
Added